lib/filewatch/tail.rb

```diff
@@ -162,6 +162,8 @@ def _open_file(path, event)
           @files[path].sysseek(stat.size, IO::SEEK_SET)
           @sincedb[sincedb_record_uid] = stat.size
         end
+      elsif event == :create && @files[path]
+        @sincedb[sincedb_record_uid] = 0
```

Comment:
Can we just add this condition as a `OR` on the line https://github.com/jordansissel/ruby-filewatch/pull/59/files#diff-d6441b4a8c5e773fbc00ca0a0cb1fcedR154 ?